### PR TITLE
fixed word count bug issue #522

### PR DESCRIPTION
**Pull request by arunavkonwar**

### CODE REVIEW

**Comment by arunavkonwar** [comment]
@avishkar58 not sure why the build is failing. I only updated the frontend.

**Comment by avishkar58** [comment]
Hey @arunavkonwar - if you merge develop in now it should fix the build problem

File: webapp/src/components/form/FormTextArea.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class FormTextArea extends React.Component {
             value={this.props.value || ""}
             onChange={this.props.onChange}
             onKeyPress={this.getWordCount}
```
Comment (avishkar58) [inline webapp/src/components/form/FormTextArea.js]: Can you remove the onKeyPress handler please, this event was deprecated: see https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event

**Comment by arunavkonwar** [comment]
@avishkar58 still the same issue with the build even after merge

**Pull request merged**